.sonarcloud.properties

```diff
@@ -0,0 +1,15 @@
+sonar.projectName=nr-site-registry
```

Comment:
Please add these to the params in the workflow `sonar_args: >`, otherwise it might be confusing, cli params vs properties

Reply:
Thanks for this, makes much more sense, done :)